docs/concepts/workloads/controllers/deployment.md

```diff
@@ -288,6 +288,8 @@ It is generally discouraged to make label selector updates and it is suggested t
 In any case, if you need to perform a label selector update, exercise great caution and make sure you have grasped
 all of the implications.
 
+Note that in API version `apps/v1beta2`, a Deployment's label selector is immutable after it gets created.
```

Comment:
Write notes with the following format:
**Note:** In API version ... {: .note}

Reply:
Done.